flashcards/backend/decks/tests/test_models_difficulty_level.py

```diff
@@ -0,0 +1,13 @@
+from ..models import DifficultyLevel
+
+
+def test_difficulty_level_gui_representation():
+    difficulty_level = DifficultyLevel(name="hard", value=1)
+    assert str(difficulty_level) == "hard"
+
+
+def test_difficulty_level_custom_save_method(db):
+    difficulty_level = DifficultyLevel(name="Hard", value=1)
+    difficulty_level.save()
+
+    assert difficulty_level.name == "hard"
```

Comment:
imo you do not check whether the object has been saved to the database and you should add more assertions 
    assert DifficultyLevel.objects.count() == 1
    assert DifficultyLevel.objects.first().name == "hard"

Reply:
I am under impression that, only thing that i change in save method of django models is name given by user, then i call on inherited django code. Therefor do i need to test that, if this is django original code?

Reply:
It has to be tested because you are overriding it, hence it is no more original django code

Reply:
and @bartoszbaran2 is right, because you have assertion for object created in your variable `difficulty_level` and of course this assertion is true. To check if sth is saved in database you have to query the database and the way Bartek showed you is good because it is querying db for the first object you just created.

Reply:
Agree, as per your recommendation I've updated test suite for this model.